packages/store/addon/-private/utils/is-non-empty-string.ts

```diff
@@ -0,0 +1,3 @@
+export default function isNonEmptyString(str?: any): str is string {
```

Comment:
I'm not sure it's appropriate for this to be an optional arg. Invoking a user-defined type guard without passing an argument doesn't make much sense to me